oxidation/libparsec/src/lib.rs

```diff
@@ -9,11 +9,12 @@ pub use libparsec_protocol as protocol;
 #[cfg(not(target_arch = "wasm32"))]
 pub use libparsec_types as types;
 
+#[cfg(feature = "test-utils")]
+pub use libparsec_testbed as testbed;
```

Comment:
why is this needed ?

we want the stuff exposed by testbed to be part of the `libparsec` public api

Reply:
We want the testbed api to be present in a production build ?

Reply:
If you can find a way that the `front generator` can use testbed only in test :man_shrugging:

Reply:
> We have similar issue in the Python binding

Except for the typing, we can include or not elements at the compilation without providing a mocked function 
see https://github.com/Scille/parsec-cloud/blob/31451b41960b03270a8d025998eedf8c6f05e327/src/lib.rs#L31-L32